Handdeck/Source/Handdeck.cpp

```diff
@@ -0,0 +1,68 @@
+#include "Handdeck.hpp"
+
+HandDeck::HandDeck(int _startSize)
+:

```

Comment:
co to za formatowanie? jeśli chcemy zrobić liste inicjalizującą to w tej samej lini:

```
: _startSize(startSize)
, _innazmienn(innna zmienna)
```